third_party/blink/renderer/core/html/media/html_media_element.idl

```diff
@@ -96,4 +96,8 @@ enum CanPlayTypeResult { "" /* empty string */, "maybe", "probably" };
     // The number of bytes consumed by the media decoder.
     [MeasureAs=PrefixedAudioDecodedByteCount] readonly attribute unsigned long long webkitAudioDecodedByteCount;
     [MeasureAs=PrefixedVideoDecodedByteCount] readonly attribute unsigned long long webkitVideoDecodedByteCount;
+
+    // Semicolon separated names of audio connectors, like "hdmi;bluetooth".
```

Comment:
Consider to use Blink Runtime Enabled Feature to guard SbPlayer specific idl interfaces.

See https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/platform/RuntimeEnabledFeatures.md;l=145?q=RuntimeEnabledFeatures%20-f:out&ss=chromium.